### ISSUE TITLE
SVG rendering broken in Sketch 85.1

### ISSUE THREAD

**Issue by macintoshhelper** [airbnb/react-sketchapp#533]
_I am..._

| -------------------------------------------------------------------------------------------------
| Reporting a bug or issue
| -------------------------------------------------------------------------------------------------

**Expected behavior:**

Should render an SVG

**Observed behavior:**

```sh
TypeError: svgImporter.graph().width is not a function. (In 'svgImporter.graph().width()', 'svgImporter.graph().width' is undefined)
```

<img width="516" alt="image" src="https://user-images.githubusercontent.com/6757532/162650257-957ca607-057d-4cbb-9bd6-6f9698d61a5f.png">

**How to reproduce:**

```sh
curl https://codeload.github.com/airbnb/react-sketchapp/tar.gz/master | tar -xz --strip=2 react-sketchapp-master/examples/basic-svg
cd basic-svg
```

**Sketch version:**

85.1

**Please attach screenshots, a zip file of your project, and/or a link to your github project**

https://github.com/airbnb/react-sketchapp/tree/master/examples/basic-svg

**Comment by macintoshhelper** [comment]
As a temporary fix (to unblock development), hardcoding the Node.js based SVG renderer for Sketch rendering works. The issue seems to be because of a Sketch private API change (`graph` property seems to be null).

**Comment by macintoshhelper** [comment]
Sketch SVG rendering seems to be fixed with my fork:

https://github.com/elemental-zcash/react-sketchapp/blob/85c5d3f5b2eade0df8d4d5f13e93a318747daf1e/src/jsonUtils/makeSvgLayer/index.sketch.ts#L13-L16

```js
  const svgNativeLayer = svgImporter.importAsLayer();
  const svgLayer = DOM.Group.fromNative(svgNativeLayer).layers[0];


  const frame = NSMakeRect(0, 0, svgLayer.frame.width, svgLayer.frame.height);
```
I’ve published an experimental forked package to unblock this issue.

```sh
npm i react-sketchapp@npm:@elemental-zcash/react-sketchapp
```

Also, not sure if worth opening a new issue, but `<Def>` support seems to be broken with the SVG Node.js renderer, hence why I fixed the Sketch API renderer rather than using the Node.js one inside the Sketch environment.
